plugins/generator-datapack-1.18.x/datapack-1.18.x/templates/recipes/smithing.json.ftl

```diff
@@ -0,0 +1,13 @@
+<#-- @formatter:off -->
+<#include "../mcitems.ftl">
+{
+  "type": "minecraft:smithing",
+  "base": {
+    ${mappedMCItemToIngameItemName(data.smithingInputStack)}
+  },
+  "addition": {
+    ${mappedMCItemToIngameItemName(data.smithingInputAdditionStack)}
+  },
+    "result": "${mappedMCItemToIngameNameNoTags(data.smithingReturnStack)}"

```

Comment:
This does not work, object is expected in this case

Reply:
When I tested, it seemed to work... I'll check that again.

Reply:
I tested the on fg gen with this variant and it did not want to work